pythonProject/and_those_names.py

```diff
@@ -0,0 +1,34 @@
+
+
+def upper_first_letter_of_name(name: str) -> str:
+    """
+    Get a name and return the name with the first letter as uppercase instead of lowercase.
+    :param name: A string containing a name.
+    :return: the name with the first letter as uppercase instead of lowercase.
+    """
+    return name[0].upper() + name[1:]
```

Comment:
you can use ,capitalize instead.
https://www.geeksforgeeks.org/string-capitalize-python/#:~:text=In%20Python%2C%20the%20capitalize(),in%20the%20string%20lowercase%20letters.

Reply:
I used capitalize, look at the second update of this branch.